### DISCUSSION TITLE
Автор забросил проект? Есть какая информация?

### DISCUSSION THREAD

**Discussion by Rvcube** [xvzc/SpoofDPI#264] [General]
Обновлений нет уже месяц. У многих спуф сбоит. Проект заброшен?

**Comment by gray1717** [comment]
уже не работает :( ютуб запинается

**Reply by DenimTornado** [reply]
window size 1 и doh на 8.8.8.8 помогли на маке

**Reply by Alex22891** [reply]
> window size 1 и doh на 8.8.8.8 помогли на маке

А какой оператор?

**Reply by DenimTornado** [reply]
билайн DC

**Reply by DenimTornado** [reply]
могу попробовать на МТС

**Reply by pinta-vodki** [reply]
И вправду. В офисе работало, а дома нет. Поменял window size с 0 на 1, заработало.

**Reply by Gen5551** [reply]
Согласен, у меня такая же ситуация. Ростелеком.
Будь проклято все 7 поколений кто основал РКН и его тупые DPI!!

**Comment by Gen5551** [comment]
> Обновлений нет уже месяц. У многих спуф сбоит. Проект заброшен?

Походу человека в тюрьму посадили...... В наше трудное время всё возможно!
